File: tests/fixtures/counter_loop.py
#!/usr/bin/env python3
# Simple counter loop for testing breakpoint + continue

def count_up(limit):
    counter = 0
    for i in range(limit):
        counter += 1  # Breakpoint here (line 7)
        print(f"Counter: {counter}")
    return counter

if __name__ == "__main__":
    result = count_up(5)
    print(f"Final: {result}")

File: tests/fixtures/counter_loop.js
#!/usr/bin/env node
// Simple counter loop for testing breakpoint + continue

function countUp(limit) {
  let counter = 0;
  for (let i = 0; i < limit; i++) {
    counter += 1;  // Line 7 - Breakpoint here
    console.log(`Counter: ${counter}`);  // Line 8
  }
  return counter;
}

const result = countUp(5);
console.log(`Final: ${result}`);
